Firebase/Core/FIROptions.m

```diff
@@ -295,6 +290,18 @@ - (void)setGoogleAppID:(NSString *)googleAppID {
 }
 
 - (NSString *)libraryVersionID {
+  static dispatch_once_t onceToken;
+  dispatch_once(&onceToken, ^{
+    // The unit tests are set up to catch anything that does not properly convert.
+    NSString *version = [NSString stringWithUTF8String:FIRCoreVersionString];
+    NSArray *components = [version componentsSeparatedByString:@"."];
+    NSString *major = [components objectAtIndex:0];
+    NSString *minor = [components objectAtIndex:1];
+    minor = ([minor length] == 2) ? minor : [NSString stringWithFormat:@"0%@", minor];
```

Comment:
We should be able to use string formatting here: `@"%02d"` would allow you to have an integer padded to two digits, becoming: `[NSString stringWithFormat:@"%02d", [minor intValue]];`. Same as below.